src/components/molecule_new_word_box/index.tsx

```diff
@@ -38,8 +38,8 @@ const NewWordBox: FC = () => {
     else setWritingMode(true)
   }, [isWritingMode, setWritingMode, onDynamicFocus])
 
-  useKeyPress(`Enter`, onHitEnter)
-  useKeyPress(`Escape`, onClickPostWordWritingModeClose)
+  useKeyPress(onHitEnter, `Enter`)
```

Comment:
This probably requires to change onHitEnter not to enter, if a dialog is open

Reply:
(should modify onHitEnter logic, where it should stop itself if a dialog is opened (like edit dialog)

Reply:
I will make an issue for this and handle in a different PR